libs/core/src/lib/quick-view/quick-view-title/quick-view-title.component.ts

```diff
@@ -1,8 +1,11 @@
-import { ChangeDetectionStrategy, Component } from '@angular/core';
+import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
 
 @Component({
     selector: 'fd-quick-view-title',
     templateUrl: './quick-view-title.component.html',
     changeDetection: ChangeDetectionStrategy.OnPush
 })
-export class QuickViewTitleComponent {}
+export class QuickViewTitleComponent {
+    /** Alignment of title. */
```

Comment:
Could you please add to the typedocs the text:
Options include 'left', 'right' and 'middle'. The default is set to 'middle'.

Thank you!